opm/simulators/aquifers/AquiferCarterTracy.hpp

```diff
@@ -92,17 +98,27 @@ class AquiferCarterTracy : public AquiferInterface<TypeTag>
     Scalar mu_w_{1}; // water viscosity
     Scalar fluxValue_{0}; // value of flux
 
+    Scalar dimensionless_time_{0};
```

Comment:
Can we just use `double`? Until we start doing `<float32>` and `<float128>` simulators on a regular basis I consider the templated scalar type to just be a unwanted complexity-

Reply:
> Can we just use `double`?

I'm just trying to stay in the style of the rest of the module.

> Until we start doing `<float32>` and `<float128>`

There are experimental versions that do that.